content/en/docs/about/community.md

```diff
@@ -5,30 +5,23 @@ weight = 10
 aliases = ["/docs/community/"]
 +++
 
-## Kubeflow Slack
-
-The Official Kubeflow Slack is used for informal discussions among users and contributors.
-
-<a href="https://communityinviter.com/apps/kubeflow/slack">
-  <button class="btn btn-primary py-2 px-5 mb-3">Click to join:<br>Kubeflow Slack</button>
-</a>
-
-| Popular Kubeflow Slack Channels |
-| --- | --- |
-| General Discussion | [#general](https://kubeflow.slack.com/archives/C7REE0ETX)
-| Feature Requests | [#feature-requests](https://kubeflow.slack.com/archives/C01A7RYEYMB)
-| Job Postings | [#job-postings](https://kubeflow.slack.com/archives/CJ9PJE5FS)
-| Kubeflow - Pipelines | [#kubeflow-pipelines](https://kubeflow.slack.com/archives/CE10KS9M4)
-| Kubeflow - Notebooks | [#kubeflow-notebooks](https://kubeflow.slack.com/archives/CESP7FCQ7)
-| Kubeflow - Training Operator | [#kubeflow-training-operator](https://kubeflow.slack.com/archives/C985VJN9F)
-| Kubeflow - Katib | [#kubeflow-katib](https://kubeflow.slack.com/archives/CE0BURK1B)
-| KServe | [#kserve](https://kubeflow.slack.com/archives/CH6E58LNP)
-| Platform - AWS | [#platform-aws](https://kubeflow.slack.com/archives/CKBA5D0MU)
-| Platform - Azure | [#platform-azure](https://kubeflow.slack.com/archives/CUW6SLCPR)
-| Platform - GCP | [#platform-gcp](https://kubeflow.slack.com/archives/CKH7V1M7F)
-| Users - China | [#users-china](https://kubeflow.slack.com/archives/C93HYNM9C)
-| Users - Korea | [#users-korea](https://kubeflow.slack.com/archives/CKPCJB9AP)
-| Users - Oceania | [#users-oceania](https://kubeflow.slack.com/archives/C023ZN1R9FC)
+## Kubeflow Slack Channels
+
+Kubeflow is part of the vibrant CNCF community. You are welcome to create your CNCF Slack account [here](https://slack.cncf.io/) and then join the following list of Kubeflow Slack channels.
```

Comment:
Do we want to keep the join button, and just replace the link with `https://communityinviter.com/apps/cloud-native/cncf` ?

Reply:
Oh I didn't know CNCF Slack had an inviter link. Although we had issues with inviters in general in the past so I am not sure if this will be stable.

Reply:
The `https://communityinviter.com/apps/cloud-native/cncf` link has worked for a number of years at this point, but also, you literally cant join by clicking `https://slack.cncf.io/`, so we have to use an inviter.

Reply:
I just realized that it's the same. https://slack.cncf.io/ will redirect to the community inviter.

Reply:
I'd like to remove the button and use the sentence instead so that it's clear that they need to join CNCF Slack first.

Reply:
Sorry, I misread the link (I thought it was a slack domain).

You are correct that `https://slack.cncf.io/` will work.

Reply:
Here is my suggested wording which I think is slightly more clear:

```suggestion
Kubeflow is part of the vibrant CNCF community, we use the [CNCF Slack](https://slack.cncf.io/) for informal discussions among users and contributors.
Please join the [Kubeflow channels](#slack-channels) to join the conversation and get help from the community.

<a href="https://slack.cncf.io/">
  <button class="btn btn-primary py-2 px-5 mb-3">Click to join:<br><b>CNCF Slack</b></button>
</a>
```

Note, I have also created a new "slack channels" heading (so we can link to it), and I have kept the button to help users know what the "action" they can take is, which will increase the percentage of people who join.